sonata-project/form-extensions/1.4/manifest.json

```diff
@@ -0,0 +1,9 @@
+{
+    "bundles": {
+        "Sonata\\Form\\Bridge\\Symfony\\SonataFormBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/",
+        "src/": "%SRC_DIR%/"
```

Comment:
Should we really copy everything in [this folder](https://github.com/sonata-project/twig-extensions/tree/1.x/src) to the user's application?

That seams wrong.

Reply:
Thank you for the feedback!
It was just a bad copy / paste from other file: https://github.com/symfony/recipes-contrib/blob/956288e30be02cb7a4d70accb6e92059da26eff7/sonata-project/admin-bundle/3.31/manifest.json#L7
I've removed this line.